.github/workflows/native.yml

```diff
@@ -0,0 +1,71 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: GraalVM Native Image Daily Continuous Integration
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - '**/pom.xml'
+      - '**/src/main/**'
+      - '!examples/**'
+      - '!shardingsphere-distribution/**'
+      - 'shardingsphere-distribution/shardingsphere-proxy-distribution/**'
+      - '!shardingsphere-test/**'
+      - '!*.md'
+
```

Comment:
if this is a 'nightly build', then the trigger for this workflow should be schedule.

Reply:
if you wanna verify something through github action , just push to your own customized branch, say, `dev`. and then add following rules in the workflow :

```
on:
  push:
    branches: [ master, dev ]
```

then whenever you push to the dev branch(no need to make Pull Request),  then this action will triggered in your own repository.

Reply:
Thanks for the reminder~